MeesGame/AI/QLearning.cs

```diff
@@ -87,9 +87,9 @@ private double EstimatedOptimalFutureValue(ILevelState s, MeesGame.PlayerAction
         /// </summary>
         /// <param name="s"></param>
         /// <param name="a"></param>
-        private void Q(ILevelState s, MeesGame.PlayerAction a)
+        private void Q(ILevelState s, MeesGame.CharacterAction a)
         {
-            Tuple<ILevelState, MeesGame.PlayerAction> StateAndAction = new Tuple<ILevelState, MeesGame.PlayerAction>(s, a);
+            Tuple<ILevelState, MeesGame.CharacterAction> StateAndAction = new Tuple<ILevelState, MeesGame.CharacterAction>(s, a);
```

Comment:
Don't change anything in the q-learning file, Mees doesn't know yet how to fix merge conflicts.